examples/util/base.js

```diff
@@ -236,12 +240,13 @@ const ssd_mobilenetv2_tflite = {
   preOptions: {
     mean: [127.5, 127.5, 127.5],
     std: [127.5, 127.5, 127.5],
-  }
-};
-
-const ssdlite_mobilenetv2_tflite = {
-  modelName: 'SSDLite MobileNet v2(TFlite)',
-  modelSize: '67.3MB',
+  },
+  intro: 'MobileNet V2 with SSD. This model is faster than Mobilenet V1 SSD at the same accuracy.',
```

Comment:
![image](https://user-images.githubusercontent.com/40754037/53549979-9beeb880-3b70-11e9-921f-6458ff781d0e.png)
![image](https://user-images.githubusercontent.com/40754037/53549995-a6a94d80-3b70-11e9-8385-6bbc1fe1228d.png)
I think SSD Mobilenet V2 is slower than V1, but have a higher accuracy.

Reply:
intro: 'MobileNet V2 with SSD. This model is slower than Mobilenet V1 SSD, but have a higher accuracy. '